canonical-websites/www.ubuntu.com
# update /desktop to vbt

## Done

* update /desktop to vbt
* made the ‘BENEFITS FOR ALL INDUSTRY SECTORS’ animations much much simpler

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/desktop)
- compare to the [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/5.%20Desktop?preview=desktop-overview.png) and the [demo](http://www.ubuntu.com-1654-desktop-overview.demo.haus/desktop)

## Issue / Card

Fixes #1654
